_tests/integration/reactnative_expo_test.go

```diff
@@ -370,12 +389,286 @@ configs:
                   COMPILER_INDEX_STORE_ENABLE = NO
                   DEVELOPMENT_TEAM = $BITRISE_IOS_DEVELOPMENT_TEAM
           - deploy-to-bitrise-io@%s: {}
+        primary:
```

Comment:
No tests are found, but *primary* workflow still created.

Reply:
The manual deploy workflow got the face-lift (removing script, certificate-and-profile-installer, etc.), but the automatic did not, is this intentional?

Reply:
Only updated the react (non-expo) deploy workflow, intentionally left the expo deploy (both manual and automatic) untouched.